code/modules/materials/Mat_MaterialProcs.dm

```diff
@@ -320,6 +320,18 @@ triggerOnEntered(var/atom/owner, var/atom/entering)
 	execute(var/atom/owner, var/mob/attacker, var/atom/attacked)
 		var/turf/T = get_turf(attacked)
 		var/mob/attacked_mob = attacked
+		if (istype(attacked, /obj)) // teleportationally unstable artifacts react to telecrystals so that scientists can determine that they are
+			var/obj/target = attacked
+			if (target.artifact?.activated && target.artifact?.teleportationally_unstable)
+				attacker.visible_message(SPAN_ALERT("[attacked] reacts to [owner]!"))
+				if (target.anchored) // anchored artis shouldn't move, but there should still be feedback for the purpose of reactivity testing
+					return
```

Comment:
maybe make the sound no matter what then

Reply:
i mean, fair enough - but it still should have some feedback. plenty of players don't read the side chat at all